src/main/resources/templates/todoAdd.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
Jak jestem w widoku dodawania to w menu nie pojawia mi się opcja `Archived`. To samo dzieje się w widoku edycji czy oglądania szczegółów konkretnego zadania.